README.md

```diff
@@ -799,6 +799,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Odin May](https://github.com/odinmay)
   - [Okechukwu Chukwude](https://github.com/okeychukwude)
   - [Okute Wonah](https://github.com/okutewonah)
+  - [Okumu Mark](https://github.com/marcokumu)
```

Comment:
Please add this above `Okute Wonah` on line `801`

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix